lib/at_cmd/at_cmd.c

```diff
@@ -7,7 +7,12 @@
 #include <logging/log.h>
 #include <zephyr.h>
 #include <stdio.h>
+#if !defined(CONFIG_NET_SOCKETS_POSIX_NAMES)
```

Comment:
I'd rather use `#if defined(CONFIG_POSIX_API)` as a factor to decide whether to use POSIX headers or not. Same in other places below.

Reply:
Yeah, that was actually my initial solution but then (don't remember why and from where ;-)) I came to conclusion that there might be a situation where posix headers are used without all stuff from POSIX_API -config. Not sure anymore if that is the real use case of these configs.

But OK, I'll change those under POSIX_API flags.